bdr-veeam/veeam-configure-backblaze-repo.ps1

```
@@ -0,0 +1,588 @@
+    Write-Host "  The B2 bucket ($BUCKET_NAME) was created but the Veeam repo was not."
+    Write-Host "  You may need to add it manually in the Veeam console."
+    Stop-Transcript
+    exit 1
+}
+
+# ============================================================
+# STORE RESULT
+# ============================================================
+
+Write-Host ""
+Write-Host "=== Complete ==="
```

PSScriptAnalyzer (1.24.0)

[warning] File 'veeam-configure-backblaze-repo.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)